src/Symfony/Bridge/Doctrine/Messenger/EntityMessagePreDispatchEvent.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Doctrine\Messenger;
+
+use Symfony\Component\Messenger\Envelope;
+
+final class EntityMessagePreDispatchEvent
```

Comment:
I don't quite understand the naming of this class. Can you explain? This seems to me like a ... `EntityMessageEvent` - but I don't understand the "Pre" part - it happens `postFlush()`.